src/main/resources/network/brightspots/rcv/GuiTabulationPopup.fxml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ RCTab
+  ~ Copyright (c) 2017-2024 Bright Spots Developers.
+  ~
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+  -->
+
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.text.*?>
+
+<Pane prefHeight="513.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/17.0.2-ea" xmlns:fx="http://javafx.com/fxml/1" fx:controller="network.brightspots.rcv.GuiTabulateController">
```

Comment:
My hesitation for making it an extra tab is noted above -- if there's software that you know of with this design, can you share it?

As one example of how a tab is more confusing UX than a modal, if this were a "Tabulate" tab:
1. User hits "Check Ballot Counts"
2. User goes to another tab, changes something
3. User returns to "Tabulate" tab

What should happen? Do we reset the state of the "Tabulate" tab? That seems like the only correct solution, but then it breaks the UX of a tabbed layout, where users expect changes on one tab to be independent of changes in another. It's a tab acting as a modal. I fear that UX is more unintuitive, but I could be convinced otherwise.

As for the logs, I can try to address it, but I'm not sure how bad it is currently: after "Check Ballot Counts," all info in the logs is also presented in the GUI. After "Tabulate," we expect the window to close anyway.

Indeed, the current releases have essentially the same behavior:
1. while the "Save" popup is open, you cannot scroll through the logs
2. while the "enter name" popup is open, you cannot scroll through the logs
3. while tabulation is in progress, each time a log output is entered, the log scrolls to the bottom -- meaning you cannot scroll through the logs until tabulation is complete

The only difference between the existing functionality and the new functionality is that you also cannot scroll through logs after hitting the "Check Ballot Counts" button.

Reply:
We had many conversations during discussion and development about a modal vs. another tab. I'm in agreement that if we continue to see user workflow issues pop up in the future we can reconsider. Between your testing and testing during the development process we've squashed the mostly likely use-case bugs 🤞